libraries/aws_conn.rb

```diff
@@ -28,4 +28,12 @@ def iam_resource
   def iam_client
     @iam_client ||= Aws::IAM::Client.new
   end
+
+  def sts_client
+    @sts_client ||= Aws::STS::Client.new
+  end
+
+  def account_id
+    sts_client.get_caller_identity[:account]
```

Comment:
I feel like this file should just expose `sts_client` and callers should be responsible for getting data out of it